src/main/java/org/inferred/freebuilder/processor/Analyser.java

```diff
@@ -312,6 +310,17 @@ private static boolean isUnderride(ExecutableElement method) {
     return userClass;
   }
 
+  private Optional<BuilderFactory> getBuilderFactory(Optional<TypeElement> builder) {
```

Comment:
I thought exactly that, then I spent ten seconds failing to think of a better prefix, and gave up. What would you call it?